nbs/api/14_quarto.ipynb

```diff
@@ -11,6 +11,14 @@
     "- order: 14"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    class SidebarYmlRemoved:
</pre><p>This should be private, so its name should start with <code>_</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>